be/src/exec/pipeline/scan_operator.cpp

```diff
@@ -98,9 +101,7 @@ bool ScanOperator::has_output() const {
 
 bool ScanOperator::pending_finish() const {
```

Comment:
The default implementation is false. and seems you should update the comments for this method in Operator parent class.